_includes/tutorials/finding-distinct/flinksql/markup/test/create-resource-populate-shipped-orders-table.sql.adoc

```diff
@@ -0,0 +1,4 @@
+Now create the sql file that will populate the deduped table: `src/test/resources/populate-deduplicated-clicks-table.sql`.
```

Comment:
capitalize SQL? or not since it's talking about the file extension... I guess I'm suggesting that it either be `Now create the SQL file that will populate ...` or `Now create the .sql file that will populate ...`